packages/strapi-design-system/src/TimePicker/TimePicker.tsx

```diff
@@ -199,24 +168,8 @@ const StyledClock = styled(Clock)`
  * TimePicker
  * -----------------------------------------------------------------------------------------------*/
 
-export interface TimePickerProps extends TimePickerInputProps, Pick<FieldProps, 'hint'> {
-  label: string;
-  labelAction?: FieldLabelProps['action'];
-}
-
-export const TimePicker = React.forwardRef<ComboboxInputElement, TimePickerProps>(
-  ({ label, error, hint, id, required, labelAction, ...restProps }, forwardedRef) => {
-    const generatedId = useId(id);
+export interface TimePickerProps extends TimePickerInputProps {}
 
-    return (
-      <Field hint={hint} error={error} id={generatedId} required={required}>
-        <Flex direction="column" alignItems="stretch" gap={1}>
-          <FieldLabel action={labelAction}>{label}</FieldLabel>
-          <TimePickerInput ref={forwardedRef} id={generatedId} error={error} required={required} {...restProps} />
-          <FieldHint />
-          <FieldError />
-        </Flex>
-      </Field>
-    );
-  },
-);
+export const TimePicker = React.forwardRef<ComboboxInputElement, TimePickerProps>(({ ...restProps }, forwardedRef) => {
+  return <TimePickerInput ref={forwardedRef} {...restProps} />;
+});
```

Comment:
Same here, I don't see any value in exporting it as a wrapper, lets just export it as is, we can rename it to just `TimePicker` if you prefer